docs/eslint.config.mjs

```diff
@@ -0,0 +1,273 @@
+import nextPlugin from '@next/eslint-plugin-next';
+import { defineConfig, globalIgnores } from 'eslint/config';
+import universeNodeConfig from 'eslint-config-universe/flat/node.js';
+import universeTypescriptAnalysisConfig from 'eslint-config-universe/flat/shared/typescript-analysis.js';
+import universeWebConfig from 'eslint-config-universe/flat/web.js';
+import lodash from 'eslint-plugin-lodash';
+import * as mdx from 'eslint-plugin-mdx';
+import tailwind from 'eslint-plugin-tailwindcss';
+import testingLibrary from 'eslint-plugin-testing-library';
+import unicorn from 'eslint-plugin-unicorn';
+
+const TAILWIND_DEFAULTS = {
+  callees: ['mergeClasses'],
+  classRegex: '^(confirmation|container|icon)?(c|C)lass(Name)?$',
+};
+
+const CORE_RULES = {
+  'prettier/prettier': 'error',
+  'no-void': ['warn', { allowAsStatement: true }],
+  'no-return-await': 'off',
+  // 'import/order': [
```

Comment:
Disabled this to prevent making changes in the docs codebase, but I think it's a valid.

Here's an example of the error:
<img width="740" alt="Screenshot 2025-04-16 at 13 50 09" src="https://github.com/user-attachments/assets/ca3729a1-6d6a-4d53-a4aa-ae21e8a79f05" />

It wants you to group the `ui` import with the `~ui` ones which seems correct to me.
<img width="727" alt="Screenshot 2025-04-16 at 13 49 59" src="https://github.com/user-attachments/assets/cb0023e8-d1a6-49a8-875e-2d833a34d305" />

Reply:
As discussed, will raise a follow-up PR to fix these issues.